easybuild/easyconfigs/s/SOAPfuse/SOAPfuse-1.27-foss-2016b-Perl-5.24.0.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PackedBinary'
+
+name = 'SOAPfuse'
+version = '1.27'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://sourceforge.net/p/%(namelower)s/wiki/Home/'
+description = """SOAPfuse is an open source tool developed for genome-wide detection of fusion
+ transcripts from paired-end RNA-Seq data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
```

Comment:
@SethosII This doesn't make sense, since nothing is getting built?

Reply:
@boegel I thought this should be there because of the dependency to `Perl` which is built with this toolchain? Or should I add the toolchain to `Perl` as a versionsuffix and also add `foss-2016b` to the versionsuffix of the EasyConfig?

Reply:
I'm referring to the `toolchainopts` enabling the use of `-fPIC` in `$CFLAGS` in the build environment.

Reply:
@boegel Oh, I see. I changed this.